File: fm-dddrive/src/main/java/io/dddrive/core/validation/model/AggregatePartValidation.java
package io.dddrive.core.validation.model;

import org.springframework.lang.Nullable;

import io.dddrive.core.validation.model.enums.CodeValidationLevel;

public interface AggregatePartValidation {

	Integer getSeqNr();

	CodeValidationLevel getValidationLevel();


	String getMessage();

	@Nullable
	String getPath();

}

File: fm-dddrive/src/main/java/io/dddrive/core/validation/model/AggregatePartValidation.kt
package io.dddrive.core.validation.model

import io.dddrive.core.validation.model.enums.CodeValidationLevel

interface AggregatePartValidation {

	val seqNr: Int

	val validationLevel: CodeValidationLevel

	val message: String

	val path: String?

}
